src/components/SuperLikeContentCard.vue

```diff
@@ -3,15 +3,20 @@
     class="super-like-content-card"
     @show="fetchContent"
   )
+    Card(v-if="isLoading")
+      Placeholder(style="width: 60%;height: 16px")
+      Placeholder(style="width: 100%;height: 16px;margin-top: 12px")
+      Placeholder(style="width: 40%;height: 16px;margin-top: 8px")
```

Comment:
why 3 placeholder instead of one?

Reply:
Each placeholder is a rectangle
<img width="496" alt="Screenshot 2020-11-03 at 6 04 13 PM" src="https://user-images.githubusercontent.com/17264716/97972149-0f91a000-1dff-11eb-961b-0aaa3e6aa051.png">